main/zoe/api/zoe.md

```diff
@@ -200,14 +245,133 @@ either the `proposal `or `payments` are empty, indicate this by
+from the `seat`'s current state, one (`getProposal()`) returns a value from an
+unchanging state (i.e. You can't change the `proposal` associated with a `seat`),
+and three (`getPayouts()`, `getPayout()`, `getOfferResult()`) extract a result.
+
+Another is 
+a `boolean` returning test, and the last attempts an action. Note that `tryExit()` 
+only works if the `seat`'s `proposal` has `OnDemand` for its `exit` clause.  
+
+- `getCurrentAllocation()`
+  - Returns: `{ Promise<Allocation> }`
+  - An `Allocation` is an `AmountKeywordRecord` of key-value pairs where
+    the key is a keyword such as `Asset` or `Price` applicable to the
+    contract. The value is an `amount` with its `value` and `brand`. 
+    
+    `Allocations` represent the `amounts` to be paid out to each `seat` on exit. 
+    Possible exits are exercising an exit condition, the contract's explicit choice, 
+    or a crash or freeze. There are several methods for finding out what `amount` 
+    a current `allocation` is.
+    
+    An `Allocation` example:
+    - ```js
+      {
+        Asset: quatloosAmountMath.make(5),
+        Price: moolaAmountMath.make(9)
+      }
+      ```
+- `getProposal()`
+  - Returns: `{ Promise<ProposalRecord> }`
+  - A `Proposal` is represented by a `ProposalRecord`. It is the rules
+    accompanying the escrow of `payments` dictating what the user expects
+    to get back from Zoe. It has keys `give`, `want`, and
+   `exit`. `give` and `want` are records with keywords as keys and
+    `amounts` as values. The proposal is a user's understanding of the
+    contract that they are entering when they make an offer. See
+    `E(zoe).offer()` for full details.  
+  - Example:
+    ```js
+    const { want, give, exit } = sellerSeat.getProposal();
+    ```
+- `getPayouts()`
+  - Returns: `{ Promise<PaymentPKeywordRecord> }`
+  - A `payout` is a `payment` that goes to a party in a successful transaction, redirecting
+    escrowed assets in accordance with the result of the transaction. Returns a record
+    of all the `payout` `payments` associated with the `seat`'s offers.
+- `getPayout(keyword)`
+  - Returns: `{ Promise<Payment> }`
+  - A `payout` is a `payment` that goes to a party in a successful transaction, redirecting
+    escrowed assets in accordance with the result of the transaction. Returns the `payout`
+    `payment` associated with the `keyword` argument.
+- `getOfferResult()`
+  - Returns: `{ Promise<OfferResult> }`
+  - The returned `OfferResult` can be literally anything. For example, in tests
+    for the Automated Refund Dapp, it's the string "The offer was accepted". In
+    the Covered Call example, it's a call option, which is an assayable `invitation`
+    to buy the underlying asset. Strings and invitations are the most common things returned.
+    The value is set by the returned result of  the `offerHandlers` method argument to `zcf.makeInvitation()`. **tyg todo: Not sure what the
+    offerHandlers look like/are represented.**
```

Comment:
```suggestion
    The value is set by the returned result of  the `offerHandlers` function passed as 
     an argument to `zcf.makeInvitation()`.
```
Aside for TYG:
We need to explain somewhere the shape of a contract.

The outer framework of a contract looks like:
``` js
const start = zcf => {
  ...
  return { creatorInvitation /* etc. */ };
};

harden(start);
export { start };
```
somewhere inside it'll define a handler:
``` js
    /** @type {OfferHandler} */
    const matchingSeatOfferHandler = matchingSeat => {  ...  };
```
and then make an invitation with it
``` js
    const matchingSeatInvitation = zcf.makeInvitation(
      matchingSeatOfferHandler,
      'matchOffer',
    );
```